js/lib/alert-bar.js

```diff
@@ -0,0 +1,16 @@
+var $ = require('jquery')
+var callData = require('./communityCall.json')
+
+function callTime () {
+  var now = new Date().getUTCHours()
+  var startTime = parseInt(callData.time, 10)
+  var dayOfWeek = new Date().getUTCDay()
+  var reminder = $('.alert-bar')
+  if ((startTime - now <= 2 && startTime - now >= 0.5) && (dayOfWeek === callData.day)) {
```

Comment:
I think you need some tests here, this will still hide the banner while the call is in progress.

Reply:
Locally the banner still appears while the call is in session on my machine- but will add test.